From 2f0fa41919c53861919f3df9d63f0ff69faeab54 Mon Sep 17 00:00:00 2001
From: Ryan QIAN <b32804@freescale.com>
Date: Fri, 26 Oct 2012 10:11:22 +0800
Subject: [PATCH 2233/2463] ENGR00231273-01 mmc: sdhci: modularize sdhci
 driver

Export sdhci_request to fix build error.

Here's the error message:
ERROR: "sdhci_request" [drivers/mmc/host/sdhci-platform.ko] undefined!

Acked-by: Dong Aisheng <b29396@freescale.com>
Signed-off-by: Ryan QIAN <b32804@freescale.com>
---
 drivers/mmc/host/sdhci.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/mmc/host/sdhci.c b/drivers/mmc/host/sdhci.c
index e8f78ae..d34486e 100755
--- a/drivers/mmc/host/sdhci.c
+++ b/drivers/mmc/host/sdhci.c
@@ -1312,6 +1312,7 @@ void sdhci_request(struct mmc_host *mmc, struct mmc_request *mrq)
 	mmiowb();
 	spin_unlock_irqrestore(&host->lock, flags);
 }
+EXPORT_SYMBOL_GPL(sdhci_request);
 
 static void sdhci_set_ios(struct mmc_host *mmc, struct mmc_ios *ios)
 {
-- 
1.7.7.4

